automated install/basic-install.sh

```diff
@@ -809,6 +809,7 @@ setDNS() {
     Quad9)
       echo "Quad9 servers"
       PIHOLE_DNS_1="9.9.9.9"
+      PIHOLE_DNS_2="149.112.112.112"
```

Comment:
Where is this IP documented? I can't find it on their website.

Reply:
https://twitter.com/Quad9DNS/status/931618659407159297

Reply:
Until they advertise it on their website, I'm unsure about adding it here.

Reply:
No worries, we can wait until their website is properly updated.

Reply:
If it's advertised on their primary Twitter account, then that's pretty dang official. I'm okay with adding this, personally.

Reply:
Hmmm, there is a possibility that the secondary isn't even filtered, if [this conversation](https://github.com/Nyr/openvpn-install/issues/379) is anything to go by